org.webonnix.beamprivacy.appdata.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally it should be shipped upstream as other distributions could benefit from it.